pkg/cmd/server/kubernetes/node/node_config_test.go

```diff
@@ -29,18 +30,29 @@ func TestKubeletDefaults(t *testing.T) {
 	expectedDefaults := &kubeletoptions.KubeletServer{
 		KubeletFlags: kubeletoptions.KubeletFlags{
 			KubeConfig: flag.NewStringFlag("/var/lib/kubelet/kubeconfig"),
-			ContainerRuntimeOptions: kubeletoptions.ContainerRuntimeOptions{
+			ContainerRuntimeOptions: config.ContainerRuntimeOptions{
 				DockershimRootDirectory:   "/var/lib/dockershim",
-				DockerExecHandlerName:     "native",
 				DockerEndpoint:            "unix:///var/run/docker.sock",
 				ImagePullProgressDeadline: metav1.Duration{Duration: 1 * time.Minute},
 				RktAPIEndpoint:            rkt.DefaultRktAPIServiceEndpoint,
 				PodSandboxImage:           "gcr.io/google_containers/pause-" + goruntime.GOARCH + ":3.0", // overridden
 				DockerDisableSharedPID:    true,
+				ContainerRuntime:          "docker",
 			},
-			CloudProvider: "auto-detect",
-			RootDirectory: "/var/lib/kubelet",
-			CertDirectory: "/var/lib/kubelet/pki",
+			CloudProvider:           "", // now disabled
+			RootDirectory:           "/var/lib/kubelet",
+			CertDirectory:           "/var/lib/kubelet/pki",
+			RegisterNode:            true,                              // this looks suspicious
```

Comment:
moved from config, defaulted to true, to flag, defaulted to true... probably need a follow up to add a parallel test method that looks at kubelet config structs with defaults applied so we can track new options and overrides.

Reply:
oh, nevermind, we have it below, where you can see this field moving out